### PR TITLE
Resolve some issues with exporting types

### PR DESCRIPTION
**Pull request by drtyh2o**
Switching from `export = {}` to `export default {}` to facilitate separately exporting and importing types. Also, some minor reorganization of the Kafka code in `mds-stream`

### CODE REVIEW

File: packages/mds-stream/kafka/agency-stream-kafka.ts [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+    limitations under the License.
+ */
+
 import { VehicleEvent, Telemetry, Device } from '@mds-core/mds-types'
 import { getEnvVar } from '@mds-core/mds-utils'
-import { AgencyStream } from '../stream-interface'
-import { KafkaStreamProducer } from './stream-producer'
+import { KafkaStreamProducer } from '.'
+
+export interface AgencyStream {
```
Comment (avatarneil) [inline packages/mds-stream/kafka/agency-stream-kafka.ts:21]: I'd like to keep this in a shared place -- I was going to reuse for the NATS client refactor that's currently in limbo.

Comment (drtyh2o) [reply packages/mds-stream/kafka/agency-stream-kafka.ts:21]: I can move it. Ideally, it ought to be in agency, but that's a bigger lift.

Comment (drtyh2o) [reply packages/mds-stream/kafka/agency-stream-kafka.ts:21]: I can move it. Ideally, it ought to be in agency, but that's a bigger lift.

Comment (drtyh2o) [reply packages/mds-stream/kafka/agency-stream-kafka.ts]: Moved to `agency-stream-interface.ts`

File: packages/mds-stream-processor/connectors/kafka-connector.ts [hunk 2]
```diff
@@ -15,16 +15,20 @@
  */
 
 import logger from '@mds-core/mds-logger'
-import stream from '@mds-core/mds-stream'
-import { StreamConsumerOptions, StreamProducerOptions } from '@mds-core/mds-stream/kafka/helpers'
+import {
```
Comment (avatarneil) [inline packages/mds-stream-processor/connectors/kafka-connector.ts:18]: How portable are imports like this? I've noticed some issues recently when doing a similar style import from an external repo.

Comment (drtyh2o) [reply packages/mds-stream-processor/connectors/kafka-connector.ts:18]: I left them that way to find out. I suspect they won't work, but they are also exported from the mds-stream/index.ts.

**Pull request merged**